public/js/reducers/sources.js

```diff
@@ -156,24 +171,30 @@ function updateTabList(state, source, tabIndex) {
  * @memberof reducers/sources
  * @static
  */
-function getNewSelectedSourceId(state, id) : ?Source {
+function getNewSelectedSourceId(state, id: any) : ?Source {
```

Comment:
@bomsy just posted this [PR](https://github.com/bomsy/debugger.html/pull/1) against this branch. In the past, I've been able to push directly to your branch, but for whatever reason I didn't have permission this time.